src/test/groovy/com/compilercharisma/chameleonbusinessstudio/integration/UserControllerITSpec.groovy

```diff
@@ -0,0 +1,134 @@
+package com.compilercharisma.chameleonbusinessstudio.integration
+
+import com.compilercharisma.chameleonbusinessstudio.dto.User
+import com.compilercharisma.chameleonbusinessstudio.enumeration.UserRole
+import org.springframework.security.test.context.support.WithMockUser
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo
+import static com.github.tomakehurst.wiremock.client.WireMock.post
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo
+import static com.github.tomakehurst.wiremock.client.WireMock.verify
+
+@WithMockUser
+class UserControllerITSpec extends BaseITSpec{
+
+    def "getAllUsers is successful"() {
+        given: "a request"
+        def request = client.get().uri("/api/v2/users/getAllUsers")
+
+        stubFor(post("/graphql/")
+                .withHeader("Authorization", equalTo("F9v4MUqdQuWAh3Wqxe11mteqPfPedUqp78VaQNJt8DSt"))
+                .withHeader("Content-Type", equalTo("application/json"))
+                .withHeader("Accept", equalTo("application/json, application/graphql+json"))
+                .withRequestBody(equalTo("{\"query\":\"query { list_UserItems { _UserItems { _id email displayName appointments } } }\"}"))
```

Comment:
Hard coded value is brittle. Maybe put this as a static member of the repository so if it changes in one place, it also changes here

Reply:
Should be now a static member on top of the class.